dockerfiles/platform-gnn-gpu/Dockerfile

```diff
@@ -20,43 +20,51 @@
 # Others may not be wanted at all in container mode - tensoflow and nvidia  example??
 
 ARG BUILD_LABEL
-ARG DOCKER_REPO
 FROM ${DOCKER_REPO}/platform-core:${BUILD_LABEL}
```

Comment:
Do we need to have 
ARG DOCKER_REPO

before using the variable here?

Reply:
fixed